endpoints/getting-started/README.md

```diff
@@ -13,6 +13,18 @@ Simply run the backend using `go run`:
 go run app.go
```

Comment:
Unrelated to this change, but would you mind deleting "Simply" above?

Reply:
Done